source/Octopus.Client/Octopus.Client.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I changed the file encoding to UTF8 with no BOM so that when we do inline edits of the file and then revert it, it doesn't leave a pending change in git.